src/AffectedUserFlow/CodeInput/CodeInputForm.tsx

```diff
@@ -64,6 +64,12 @@ const CodeInputForm: FunctionComponent = () => {
       if (response.kind === "success") {
         const token = response.body.token
         const exposureKeys = await strategy.getExposureKeys()
+
+        if (exposureKeys.length === 0) {
```

Comment:
I stopped the code here, but the verification server seems ok with having an empty set of keys, should we still run it down and then skip the publish step? Not 100% familiar with the verification server inner works.

Reply:
This seems okay for now to me, but imo it would be better to have the api handle these edge cases rather than the client.

Reply:
I was feeling the same way but the way we are handling the error messages gives little to no detail to the client-side, I created a [ticket] for it and going to try it in android to see what's the level of detail we are getting. 

[ticket]: https://trello.com/c/KeLmnh0c